src/test/rustdoc-json/generic-associated-types/gats.rs

```diff
@@ -0,0 +1,19 @@
+// ignore-tidy-linelength
+
+#![no_core]
+#![feature(generic_associated_types, lang_items, no_core)]
+
+#[lang = "sized"]
+pub trait Sized {}
+
+pub trait LendingIterator {
+    // @count gats.json "$.index[*][?(@.name=='Item')].inner.generics.params" 1
+    // @is gats.json "$.index[*][?(@.name=='Item')].inner.generics.params[*].name" \"\'a\"
```

Comment:
You can use `-` for the filename to use the same as the previous assert (just like in html)

Reply:
cool!